meta-dts-distro/recipes-dts/reports/dasharo-hcl-report/dasharo-hcl-report

```diff
@@ -168,11 +168,24 @@ update_result "Input bus types"  logs/ioports.err.log
 printf '################################                                   |\r'
 
 # echo "Trying to read firmware image with flashrom..."
-# On MSI boards some regions may be not available so we need to use specific
-# ones
-if [ "$BOARD_VENDOR" == "Micro-Star International Co., Ltd." ] && [ "$SYSTEM_MODEL" == "MS-7E06" ]; then
-    FLASHROM_ADD_OPT_READ="--ifd -i fd -i me -i bios"
+# Some regions may be not available so we need to use specific regions to read
+check_intel_regions
+if [ $BOARD_HAS_FD_REGION -eq 0 ]; then
+  # Use safe defaults. Descriptor may contain additional regions not detected
+  # by flashrom and will return failure when attempted to be read. BIOS and
```

Comment:
Lenovo M900 Tiny has some write-only ranges in the BIOS region, which is infuriating, probably a corner case